redash/query_runner/python.py

```diff
@@ -144,7 +153,9 @@ def execute_query(self, data_source_name_or_id, query):
         except models.DataSource.DoesNotExist:
             raise Exception("Wrong data source name/id: %s." % data_source_name_or_id)
 
-        data, error = data_source.query_runner.run_query(query)
+        query_runner = get_query_runner(data_source.type, data_source.options)
```

Comment:
Really? When did it go? Is it in a specific branch?

I'll fix it.

On Thu, Feb 25, 2016, 23:59 Arik Fraimovich <notifications@github.com>
wrote:

> In redash/query_runner/python.py
> <https://github.com/getredash/redash/pull/854#discussion_r54172274>:
>
> > @@ -144,7 +153,9 @@ def execute_query(self, data_source_name_or_id, query):
> >          except models.DataSource.DoesNotExist:
> >              raise Exception("Wrong data source name/id: %s." % data_source_name_or_id)
> >
> > -        data, error = data_source.query_runner.run_query(query)
> > +        query_runner = get_query_runner(data_source.type, data_source.options)
>
> get_query_runner doesn't exist anymore.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/getredash/redash/pull/854/files#r54172274>.
>